unbxdAutosuggest.js

```diff
@@ -411,6 +411,11 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 						return;
 					if (dataValue) {
 						var query = dataValue + (dataFiltername != '' ? ':' + dataFiltername + ':' + dataFiltervalue : '')
+						// updating product header while hovering on suggestions
+						if (self.options.filtered && self.options.popularProducts.header) {
+							self.options.popularProducts.header = self.options.popularProducts.header +
+								'&nbsp;in&nbsp;<span>' + dataValue + '</span>';
```

Comment:
Not generalised, the template of header is hard coded(Taking about `in` aspect).. How can we make more generic?

Reply:
@anantheshadiga solved this in `4c4fe11`. 
Now providing option for handleBar template in popularProduct header.